apps/www/registry/stories/select.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { Meta, StoryObj } from "@storybook/react"
+
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectSeparator,
+  SelectTrigger,
+  SelectValue,
+} from "@/registry/default/ui/select"
+
+const meta: Meta<typeof Select> = {
+  title: "ui/Select",
+  component: Select,
+  tags: ["autodocs"],
+  argTypes: {},
+}
+export default meta
+
+type Story = StoryObj<typeof Select>
+
+export const Base: Story = {
+  render: (args) => (
+    <Select {...args}>
+      <SelectTrigger className="w-[180px]">
```

Comment:
From the Radix documentation on for Select ([docs](https://www.radix-ui.com/primitives/docs/components/select)) it seems like this shouldn't be needed: 
>Adheres to the [ListBox WAI-ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox).  
><sub>[link to text](https://www.radix-ui.com/primitives/docs/components/select#:~:text=Adheres%20to%20the%20ListBox%20WAI%2DARIA%20design%20pattern.)</sub>

Though its possible to add a `<Label />` to the Select, I think for this simple of an example its more to showcase the interaction with the select+dropdown.  This could maybe be brought up with the documentation and example?